### PR TITLE
[CFM-1229] contact form within group, for group members.

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #967 

https://issuetracker.amplexor.com/jira/browse/CFM-1229

### CODE REVIEW

**Comment by anvmn** [comment]
selection_091

**Comment by anvmn** [comment]
selection_092

**Comment by anvmn** [comment]
@ordavidil 
Travis didn't run. Coder passes.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.default_elysia_cron_rules.inc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <?php
-
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.default_elysia_cron_rules.inc]: revert this

**Comment by anvmn** [comment]
@ordavidil 
Fix to support GA.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user/includes/c4m_user.theme.inc [hunk 2]
```diff
@@ -353,3 +353,54 @@ function theme_c4m_user_contact_link(array $field) {
...
+    return NULL;
+  }
+
+  // If requested user has been blocked, do not allow users to contact them.
+  if (empty($account->status)) {
+    return NULL;
+  }
+
+  $group = c4m_og_current_group();
+  if ($group) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user/includes/c4m_user.theme.inc]: remove this condition, seems worthless

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user/includes/c4m_user.theme.inc]: This field is part of User 'Block List' display.
This display if used also at /people page, which is not within OG context.
If OG context is not checked, og_get_user_roles('node', $group->nid, $user->uid)) will be activated, with $group->nid = NULL, which causes an error.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user/includes/c4m_user.theme.inc]: so change it to:
```PHP
if (!$group) {
  return NULL;
}
```

**Comment by ordavidil** [comment]
@anvmn 
Please fix the PR by @mfernea advise:
>Looks ok. I only have an issue with the naming "c4m_user_block_contact" / "C4M User block contact link". I would use something with group instead of block. Isn't it better?

**Pull request merged**